synapse/lib/storm.py

```diff
@@ -2007,11 +2007,12 @@ async def iterStormPodes(self):
         dopath = nodeopts.get('path', False)
         dolink = nodeopts.get('links', False)
         virts = nodeopts.get('virts', False)
+        verbs = nodeopts.get('verbs', True)
         show_storage = nodeopts.get('show:storage', False)
 
         async for node, path in self.execute():
 
-            pode = node.pack(dorepr=dorepr, virts=virts)
+            pode = node.pack(dorepr=dorepr, virts=virts, verbs=verbs)
```

Comment:
Quick discussion item from the ticket: do we even want this to be an option, or should we always include it?

Reply:
Discussion result: leave option to disable